library/src/main/java/com/airbnb/android/airmapview/AirMapInterface.java

```diff
@@ -194,4 +195,16 @@
 
   /** Sets the type of map tiles that should be displayed */
   void setMapType(MapType type);
-}
\ No newline at end of file
+
+  /**
+   * Construct a polygon with the given {@link LatLng} points
+   */
+  <T> void addPolygon(AirMapPolygon<T> polygon);
```

Comment:
the javadocs comment seems outdated. You refer to a LatLng but it takes an AirMapPolygon